castai/resource_service_account.go

```diff
@@ -278,3 +295,276 @@ func getOrganizationID(ctx context.Context, data *schema.ResourceData, meta inte
 
 	return organizationID, nil
 }
+
+func resourceServiceAccountKey() *schema.Resource {
```

Comment:
If we treat this as separate resource, why not make a separate file `resource_service_account_key.go` to keep the files smaller?